### PR TITLE
Add section-5 (Guest Review)

### PR DESCRIPTION
**Pull request by katsu223**
[Deploy link](https://katsu223.surge.sh)

### CODE REVIEW

File: resources/css/index.css [hunk 1]
```diff
@@ -330,3 +335,53 @@ div {
...
+
+.fourth-section {
+  background: url("../images/background/section-5.jpg") no-repeat;
+  text-align: center;
+  padding: 100px 0;
+}
+
+.review {
+  display: inline-block;
+  background: rgba(0, 0, 0, 0.8);
```
Comment (magicsd) [inline resources/css/index.css]:
```suggestion
  background-color: rgba(0, 0, 0, .8);
```
And check properties sorting

File: index.html [hunk 2]
```diff
@@ -212,6 +212,15 @@ <h3 class="dish-name">
         </div>
       </div>
     </section>
+    <section class="fourth-section">
+      <div class="review">
+        <h2 class="review-title">Guest Reviews</h2>
+        <img class="divider" src="resources/images/background/white-divider.png">
+        <p class="review-quote"><span class="quote-symbol">“</span>If you’ve been to one of our restaurants, you’ve seen – and tasted – what keeps our customers coming back for more. Perfect materials and freshly baked food, delicious Lambda cakes,  muffins, and gourmet coffees make us hard to resist! Stop in today and check us out! <br>
+        <span class="signature">- food inc, New York</span>
```
Comment (magicsd) [inline index.html]:
```suggestion
        <div class="signature">- food inc, New York</div>
```
And get this element out to parent.

File: resources/css/index.css (hunk 1 above)
Comment (katsu223) [reply resources/css/index.css]: fixed

File: index.html (hunk 2 above)
Comment (katsu223) [reply index.html]: fixed

File: resources/css/index.css (hunk 1 above)
Comment (magicsd) [reply resources/css/index.css]: Disagree

Comment (katsu223) [reply resources/css/index.css]: fixed

**Pull request merged**